src/tools/dev/scripts/bv_support/bv_qwt.sh

```diff
@@ -71,8 +71,8 @@ function bv_qwt_print
 
 function bv_qwt_print_usage
 {
-    printf "%-20s %s [%s]\n" "--qwt" "Build with Qwt" "$DO_QWT"  
-    printf "%-20s %s [%s]\n" "--alt-qwt-dir" "Use Qwt from an alternative directory"
+    printf "%-20s %s\n" "--qwt" "Build with Qwt"
+    printf "%-20s %s []\n" "--alt-qwt-dir <path>" "Use Qwt from an alternative directory"
```

Comment:
Added '<path>' to show an argument is required with --alt-qwt-dir. Removed the %s from [], but left the brackets to show there is no default for this option.
@brugger1, if you approve, I will make similar changes in the other bv files.

Reply:
I think the brackets should be removed as well. It doesn't apply and potentially could be confusing. Also, I assume there is no longer any reason to specify qwt. If so, we should probably deprecate it at some point, so showing it in the list of options seems like a bad idea. Removing the option for a bug release is probably a bad idea, since it may break existing invocations (assuming unsupported options cause build_visit to exit)

Reply:
No reason to specify qwt ? Why ? Because it is required? All required libraries are listed the same way.  Should all required libraries lose the --libname usage text and only have the --alt-lib-dir if allowed?

Reply:
Since you removed "DO_QWT" from the print statement, my assumption was that the user no longer needed to specify --qwt, since using --qt would enable it. I now believe this is incorrect. I now believe the user still has to specify --qwt with --qt in order to build VisIt. Sorry about that. I should have been more thorough in researching things before commenting. Please ignore my comment about removing --qwt.

With my new understanding, I now believe that --qwt does is fact have a default value and it is off. If this is the case, then "DO_QWT" should be added back as the default value. I tried the following things with build_visit3_0_1:

./build_visit3_0_1 --no-visit --no-thirdparty --qt

and it happily went about building Qt. I also tried

./build_visit3_0_1 --no-visit --no-thirdparty --qwt

and it gave me an error saying that I need qt to build qwt.

Reply:
DO_QWT is only off when --no-thirdparty is specified, same as all required libs.
There error when you only used --qwt is due to qwt's dependence on qt.

Reply:
BTW, if you think this is getting too much for the RC, I will gladly move the branch to develop instead.